_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -52,11 +53,11 @@ function add_actions() {
 
 		// Attach hooks common to all Jetpack admin pages based on the created
 		// hook
-		add_action( "load-$hook",                array( $this, 'admin_help'      ) );
-		add_action( "load-$hook",                array( $this, 'admin_page_load' ) );
-
-		add_action( "admin_print_styles-$hook",  array( $this, 'admin_styles'    ) );
-		add_action( "admin_print_scripts-$hook", array( $this, 'admin_scripts'   ) );
+		add_action( "load-$hook", array( $this, 'admin_help' ) );
+		add_action( "load-$hook", array( $this, 'admin_page_load' ) );
+		add_action( "admin_head-$hook", array( $this, 'admin_head' ) );
```

Comment:
Is this used anywhere?

Reply:
Good catch! The only reference I found was in this old commit 
https://github.com/Automattic/jetpack/commit/7436091201880f5163d1b5c1deba5f18e7f9411c#diff-46a389aadbd83b4fa94c516047c94018R36 five years ago.
It's used to fire the action `'jetpack_module_configuration_head_' . $_GET['configure']` 
I couldn't find that action in our current code so I'll remove this.